README.md

```diff
@@ -10,3 +10,15 @@ Run `grunt` for building and `grunt serve` for preview.
 ## Testing
 
 Running `grunt test` will run the unit tests with karma.
+
+## Git-hooks Installation and Usage
```

Comment:
may be worth adding here that `brew install git-hooks` looks like it works for OS X, which is handy?